ctapipe/containers.py

```diff
@@ -1158,7 +1163,7 @@ class SimulatedShowerDistribution(Container):
 
     default_prefix = ""
 
-    obs_id = Field(-1, description="links to which events this corresponds to")
+    obs_id = obs_id_field()
     hist_id = Field(-1, description="Histogram ID")
```

Comment:
Ignore the first comment. I don't know what the data model requires here

Reply:
> Also shower_primary_id in the SimulatedShowerContainer defaults to -1

So the default is a positron, which is kind of not intended.. but also something we do not simulate I think, so not that bad.

But yes, we should switch to something like `np.iinfo(np.int16).max` if that is not a real particle...

Reply:
Yes, I think setting it to the max is fine - clearly we will never simulate that :-)  But it is possible we would simulate positrons at some point, so better to avoid -1

Reply:
Done